README.md

```diff
@@ -21,8 +21,7 @@ scales horizontally by adding units. It supports Juju storage.
 The snap uses strict confinement with the removable-media plug to support
 storage.
 
-
-    juju deploy ./charm/builds/kafka
+    juju deploy ./charm/builds/kafka --constraints "mem=64G cores=4"
```

Comment:
Pretty heavy requirements without any reason or justification. I'd remove from this specific command example as it's maybe off-topic -- here we're just introducing how to get a basic working deployment. However if you have a good guideline in mind, you may add a section below recommending memory/CPU resources and various constraint settings. Not necessary at this point, and probably needs thoughtful research and benchmarking.